src/cli/cluster/base_path_proxy.js

```diff
@@ -18,123 +18,47 @@
  */
 
 import { Server } from 'hapi';
-import { notFound } from 'boom';
-import { map, sample } from 'lodash';
-import { map as promiseMap, fromNode } from 'bluebird';
-import { Agent as HttpsAgent } from 'https';
-import { readFileSync } from 'fs';
-
-import { setupConnection } from '../../server/http/setup_connection';
-import { registerHapiPlugins } from '../../server/http/register_hapi_plugins';
+import { createBasePathProxy } from '../../core';
 import { setupLogging } from '../../server/logging';
 
-const alphabet = 'abcdefghijklmnopqrztuvwxyz'.split('');
+export async function configureBasePathProxy(config) {
+  // New platform forwards all logs to the legacy platform so we need HapiJS server
+  // here just for logging purposes and nothing else.
+  const server = new Server();
```

Comment:
Can you explain this point a bit more? Why?

Reply:
Yeah, it will be handled in the scope of https://github.com/elastic/kibana/issues/13241 probably. And I don't want to use `even-better` in the new platform even the latest version.